src/sentry/api/serializers/models/user.py

```diff
@@ -34,15 +46,17 @@ def get_attrs(self, item_list, user):
             )
         }
         identities = self._get_identities(item_list, user)
+        emails = self._get_useremails(item_list, user)
 
         authenticators = Authenticator.objects.bulk_users_have_2fa([i.id for i in item_list])
 
         data = {}
         for item in item_list:
             data[item] = {
                 'avatar': avatars.get(item.id),
-                'identities': identities.get(item.id),
+                'identities': identities[item.id],
```

Comment:
Wait — can't this one *actually* be unset, due to the filter on the item list on L19?

I'm also not sure what initiated this change, but maybe that same check should be applied here?